response_operations_ui/controllers/uaa_controller.py

```diff
@@ -71,11 +71,10 @@ def login_admin():
         abort(response.status_code)
 
 
-def get_user_by_email(email: str, access_token=None) -> dict | None:
+def get_user_by_filter(user_filter: str, access_token=None) -> dict | None:
```

Comment:
Instead of enter another load of boiler plate using the filter for something else I rationalised the call